pkg/apis/kops/componentconfig.go

```diff
@@ -384,6 +384,9 @@ type KubeAPIServerConfig struct {
 	KubeletPreferredAddressTypes []string `json:"kubeletPreferredAddressTypes,omitempty" flag:"kubelet-preferred-address-types"`
 
 	StorageBackend *string `json:"storageBackend,omitempty" flag:"storage-backend"`
+
+	AuthorizationMode          *string `json:"authorizationMode,omitempty" flag:"authorization-mode"`
+	AuthorizationRBACSuperUser *string `json:"authorizationRbacSuperUser,omitempty" flag:"authorization-rbac-super-user"`
```

Comment:
note that the superuser flag was removed post-1.5. not sure you want to add it just for the 1.5 release.